### ISSUE TITLE
In stage 2, training loss is rising

### ISSUE THREAD

**Issue by yc-cui** [MIV-XJTU/ARTrack#63]
This is the logs for stage 2:
<img width="1390" alt="image" src="https://github.com/MIV-XJTU/ARTrack/assets/68642492/82e9837c-26c1-4f38-91d1-2aebda555c43">

<img width="1393" alt="image" src="https://github.com/MIV-XJTU/ARTrack/assets/68642492/a8b8231e-a1ba-4ee7-81cb-f47d847b2daf">

Stage 1 training is normal:
<img width="1374" alt="image" src="https://github.com/MIV-XJTU/ARTrack/assets/68642492/bb008ae4-dcc2-42a0-bc18-dbf966255049">

I only changed the number of GPUs from 4 to 2.

**Comment by AlexDotHam** [comment]
The 2 stage training is unstable because the sample of a video sequence may be causes diverse difficulties in training. It is normal if you train more than 5 epochs, the average loss will slowly decrease.

**Comment by yc-cui** [comment]
OK，I will check the results when training is finished.

**Comment by yc-cui** [comment]
> If you need my training log, i am reproduce my base_256_got model, i will release that maybe 2 hours later?

That's very kind of you for doing so.
